docs/release/trg-4/trg-4-4.md

```diff
@@ -0,0 +1,18 @@
+---
+title: TRG 4.04 - Image signing
+---
+
+| Author               | Status | Created      | Post-History    |
+|----------------------|--------|--------------|-----------------|
+| Catena-X System Team | Draft  | 13-Sept-2022 | n/a             |
+| Catena-X System Team | Active | 04-Jan-2023  | Initial release |
+
+## Description
+
+Image signing is a process where the publisher use the digital signatures where push images,and it allows verification from image consumers.The publisher can manually sign the image or have the signing process a part of their software release process. Check [here](https://docs.docker.com/engine/security/trust/#signing-images-with-docker-content-trust) to learn how it works.
```

Comment:
A space is missing between "consumers.The"

Also it reads wrong:
"where push images,and" did you mean "when pushing images, and"?

Reply:
changed